CONTRIBUTING.md

```diff
@@ -35,13 +35,14 @@ See our [Macaron Style Guide](./docs/source/pages/developers_guide/style_guide.r
 
 1. Ensure there is an issue created to track and discuss the fix or enhancement
    you intend to submit.
-2. Fork this repository.
+2. Fork this repository including the `staging` branch. In Macaron, the `staging` branch is the active development branch and contains the most recent changes. Make sure to create a branch from the `staging` branch and not `main`.
```

Comment:
```suggestion
2. Fork this repository including the `staging` branch. In Macaron, the `staging` branch is the active development branch and contains the most recent changes. Make sure to create your working branch from the `staging` branch and not `main`.
```

Reply:
I prefer to keep it simple and avoid using the `working` branch. I decided to move the comment to the next bullet point. See [c947861](https://github.com/oracle/macaron/pull/658/commits/c947861c7c1d9307456a4a188e7b267e50852fbc).